package.json

```
@@ -18,7 +19,8 @@
     "watch": "npm run build:dev -- --watch",
     "release-zip": "npm run build && npm run zip",
     "test": "npm run build:production -- --display=none && bundlesize",
-    "test:visualtest": "npm run build:static && start-server-and-test storybook:vrt http-get://localhost:9001 backstopjs",
+    "pretest:visualtest": "npm run build:storybook --silent -- --quiet --loglevel warn",
```

Then these are no longer needed here.
```suggestion
    "pretest:visualtest": "npm run build:storybook",
```